module2/collections.py

```diff
@@ -0,0 +1,38 @@
+import random
+import string
+
+# Part 1
+# Create list if random number of dicts.
+random_dict = dict()
+random_list = []
+for i in range(random.randint(2, 10)):
+    for j in range(random.randint(0, 10)):
```

Comment:
Max amount of keys in each dict should be the same as amount of lowercase eng letters

Reply:
Fixed.